### PR TITLE
[Reviewer: Ellie] Add a config flag to always behave as though the HSS had sent Include…

### PR DESCRIPTION
**Pull request by rkd-msw**
…RegisterRequest and IncludeRegisterResponse in the XML

Some HSSes don't allow you to set these fields, but some TASes require them. This pull request allows you to set `force_third_party_reg_body=Y` in the config, which always behaves as if the HSS had set it.

This makes Clearwater a useful bridge for interoperating between these sort of HSSes and these sort of TASes.

### CODE REVIEW

File: sprout/registration_utils.cpp [hunk 1]
```diff
@@ -79,10 +83,16 @@ void send_register_to_as(pjsip_rx_data* received_register,
                          const std::string&,
                          SAS::TrailId);
 
+void RegistrationUtils::init(SNMP::RegistrationStatsTables* _third_party_reg_stats_tables,
+                             bool _force_third_party_register_body)
+{
+  third_party_reg_stats_tables = _third_party_reg_stats_tables;
```
Comment (eleanor-merry) [inline sprout/registration_utils.cpp]: Not sure about the names and underscores here

File: sprout-base.root/etc/init.d/sprout [hunk 2]
```diff
@@ -165,6 +165,7 @@ get_daemon_args()
         [ "$enforce_user_phone" != "Y" ] || user_phone_arg="--enforce-user-phone"
         [ "$enforce_global_only_lookups" != "Y" ] || global_only_lookups_arg="--enforce-global-only-lookups"
         [ "$override_npdi" != "Y" ] || override_npdi_arg="--override-npdi"
+        [ "$force_third_party_reg_body" != "Y" ] || force_3pr_body_arg="--force-3pr-body"
```
Comment (eleanor-merry) [inline sprout-base.root/etc/init.d/sprout]: You'll add this to the config options guide too, yes?

**Comment by eleanor-merry** [comment]
LGTM

File: sprout/registration_utils.cpp (hunk 1 above)
Comment (rkd-msw) [inline sprout/registration_utils.cpp]: `third_party_reg_stats_tables_arg` etc. instead?

Comment (eleanor-merry) [inline sprout/registration_utils.cpp]: Sure

**Pull request merged**